cmd/create.go

```diff
@@ -475,6 +475,14 @@ func (c createConfig) prompt(client *fn.Client) (createConfig, error) {
 		return c, err
 	}
 
+	executable, err := os.Executable()
+	if err != nil {
```

Comment:
This will return an absolute path, and can even include `../` if the user is running the command outside of PATH (such as when developing).  Since we want to tell the current user exactly what command they can run again, presuming their current environment, you can simply use the first argument: `os.Args[0]`.

Reply:
or maybe use `cmd.Root().Name()`